sdk/storage/azure-storage-file-share/src/test/resources/session-records/ModelTestsNTFSToAttributes0.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think the capitalization on the file name is causing playback failures in Linux, I think `NTFS` -> `Ntfs` is needed.

Reply:
re-recorded and pushed that. I think Windows doesn't replace the file name if the caps are different?